src/detectors/BEMC/RawCalorimeterHit_factory_EcalBarrelScFiRawHits.h

```diff
@@ -36,7 +36,7 @@ class RawCalorimeterHit_factory_EcalBarrelScFiRawHits : public JFactoryT<edm4hep
 
         // Set default values for all config. parameters in CalorimeterHitDigi algorithm
         m_input_tag = "EcalBarrelScFiHits";
-        m_tRes = 0.0 * ns;
+        u_eRes = {0.0 * MeV};
         m_tRes = 0.0 * ns;
         m_capADC = 8096;
```

Comment:
This should be `'BEMC:EcalBarrelScFiRawHits:capacityADC',                   '16384',`